common/device-model.cpp

```diff
@@ -2741,13 +2744,16 @@ namespace rs2
                 {
                     bool stop_recording = false;
 
-                    ImGui::SetCursorPos({ windows_width - 35, pos.y + 3 });
+                    ImGui::SetCursorPos({ windows_width - 42, pos.y + 3 });
                     ImGui_ScopePushFont(window.get_font());
 
                     ImGui_ScopePushStyleColor(ImGuiCol_Button, sensor_bg);
                     ImGui_ScopePushStyleColor(ImGuiCol_ButtonHovered, sensor_bg);
                     ImGui_ScopePushStyleColor(ImGuiCol_ButtonActive, sensor_bg);
 
+                    int font_size = window.get_font_size();
```

Comment:
What button is this?

Reply:
![image](https://github.com/IntelRealSense/librealsense/assets/26552140/97220b82-9e17-4e70-a8ca-f3f92ed4693a)
All switches there.